prow/config/jobs/envoy-1.10.yaml

```diff
@@ -0,0 +1,66 @@
+org: istio
+repo: envoy
+branches: [release-1.10]
+image: piotrsikora/envoy-build-ubuntu@sha256:e2775df4cf57e86920ca39886a877b4ccc12dbcfaa2a7c2a804f4cb83c797952
```

Comment:
I expect we can update this at some point as well, but not sure. @PiotrSikora

Reply:
Uhm, this should match version from `.bazelrc` file in Envoy repo for a given version. Whatever is is my fork is from early Wasm days and extremelly outdated.